cloud_testing/kotlin_poc/src/test/kotlin/ftl/config/YamlConfigTest.kt

```diff
@@ -59,6 +59,7 @@ class YamlConfigTest {
         assert(config.waitForResults, true)
         assert(config.testMethods, listOf(testName))
         assert(config.limitBreak, false)
+        assert(config.devices, listOf(Devices("Nexus6"), Devices(version = "23", orientation = "landscape"), Devices("shamu", "22", "zh_CN", "default")))
```

Comment:
I think we'd want to copy `flank.yml` into a `fixtures` dir under `/Users/medwards/code/flank/cloud_testing/kotlin_poc/src/test/kotlin/ftl`. Then we don't have to modify the `flank.yml` we use for running the sample app with the `flank.yml` used by the unit tests.